server/Services/Internal/BookingService.cs

```diff
@@ -180,7 +180,7 @@ public static List<string> ValidateUserBookingRequest(CreateBookingRequest booki
         }
 
 
-        private static List<string> ValidateUserDeleteBookingRequest(Booking? booking, UserClaims user)
+        public static List<string> ValidateUserDeleteBookingRequest(Booking? booking, UserClaims user)
```

Comment:
private methods can be unit tested indirectly through the public methods that call them. try not change the private method to public just for testing

Reply:
Jeg tror fremdeles den burde testes seperat. 
Jeg så først på å gjøre det indirekte gjennom CreateBookingAsync. Det ble veldig mye mer rotete, måtte mocke repo og videre. 

Jeg synes å gjøre det på denne måten er ryddigere, og at request blir validert riktig føles som noe viktig å teste